packages/botonic-plugin-contentful/src/contentful/manage/manage-asset.ts

```diff
@@ -51,6 +51,51 @@ export class ManageContentfulAsset {
     await this.writeAsset(context, oldAsset)
   }
 
+  async removeAsset(context: ManageContext, assetId: AssetId): Promise<void> {
+    const environment = await this.environment
+    try {
+      const asset = await environment.getAsset(assetId.id)
+      if (asset.isPublished()) await asset.unpublish()
+      await asset.delete()
+    } catch (e) {
+      throw new CmsException('ERROR while deleting asset', e)
```

Comment:
when giving information about an error, it's important to have a quick way to identify the resource in question. I would return `ERROR while deleting asset with ID: '${assetId}' .`

Reply:
@AinaVendrell here you can do new CmsException('ERROR while deleting asset', e, assetId)